### PR TITLE
Identify button for waysofbeingindex

### PR DESCRIPTION
**Pull request by Zmuse001**
Changes
created identify button for waysofbeingindex and linked it to identification form

Link to Issue
https://app.asana.com/0/1206510424537042/1207090158804667/f

Comments, Notes, or Questions
This section is if you want to share something related to the PR with the reviewer.

Related Screenshots
Screenshot 2024-06-05 at 11 59 06 AM

Before you submit for review, Did you...
 Added relevant tests (if applicable)
 Linked Asana ticket to PR
 Added comments, notes, or questions for reviewwer
 Added relevant screenshots
 Ensured ticket title is clear and concise(i.e. 'Donation Form Feature' instead of 'sfg/chels/donation-form'

### CODE REVIEW

File: app/views/ways_of_being/index.html.erb [hunk 1]
```diff
@@ -2,11 +2,14 @@
...
         <h1 class="text-center font-bold lg:text-2xl py-4 border-b border-gray-600">Behaviors</h1>
         <ul class="px-4">
-            <% @ways_of_being.each do |way_of_being| %>
-            <li class="bg-white py-2 border-b border-gray-400 text-gray-400">
-                <%= way_of_being.name %>   
-            </li>
+        <% @ways_of_being.each do |way_of_being| %>
+            <li class="flex w-full justify-between bg-white py-2 border-b border-gray-400 text-gray-400">
+                <div><%= way_of_being.name %></div>
+                <%= link_to new_identification_path(student_way_of_being_id: way_of_being), class: "strategy-btn border-2 border-gray-600 hover:bg-green-600 text-black px-2 rounded flex items-center space-x-1 text-lg" do %>
```
Comment (edithemmings) [inline app/views/ways_of_being/index.html.erb:8]: ^way of being is getting passed in to student way of being.

**Review by fouziiah — COMMENTED** [review]
Screenshot 2024-06-05 at 12 10 32 PM

This Error is displaying, when I click on the btn

File: app/helpers/application_helper.rb
Comment (Zmuse001) [inline app/helpers/application_helper.rb:1]: not used, can remove

File: app/views/ways_of_being/index.html.erb [hunk 2]
```diff
@@ -2,11 +2,14 @@
...
-            <% @ways_of_being.each do |way_of_being| %>
-            <li class="bg-white py-2 border-b border-gray-400 text-gray-400">
-                <%= way_of_being.name %>   
-            </li>
+        <% @student_ways_of_being.each do |student_way_of_being| %>
+            <li class="flex w-full justify-between bg-white py-2 border-b border-gray-400 text-gray-400">
+                <div><%= student_way_of_being.way_of_being.name %></div>
+                <%= link_to new_identification_path(student_way_of_being_id: student_way_of_being.id), class: "strategy-btn border-2 border-gray-600 hover:bg-green-600 text-black px-2 rounded flex items-center space-x-1 text-lg" do %>
+                <span>Identify</span>
+                <img src="<%= asset_path("right_arrow.svg") %>" >
```
Comment (Zmuse001) [inline app/views/ways_of_being/index.html.erb:10]: missing li and tabbing fixes

File: app/helpers/application_helper.rb
Comment (Zmuse001) [reply app/helpers/application_helper.rb:1]: done

**Review by camckin10 — APPROVED** [review]
Screen Shot 2024-06-06 at 9 00 33 AM
Buttons appear, and after clicking button, I am directed to the Identifications form. Great!

**Review by KNAthari — APPROVED** [review]

**Pull request merged**